src/ui/pages/ConnectionDetails/ConnectionDetails.tsx

```diff
@@ -81,6 +81,9 @@ const ConnectionDetails = ({
       if (connectionDetails.notes) {
         setNotes(connectionDetails.notes);
       }
+      if (connectionDetails.historyItems) {
```

Comment:
Curious here if the UI will work if we have `[]` and just remove this if statement, and the previous if statement for `.notes`

Reply:
It works because it is set to default `[]` when missing

Reply:
any issues here or all ok?